lib/workers/global/ssh_socket.ts

```diff
@@ -0,0 +1,40 @@
+import type { HostRule } from '../../types';
+import * as cp from 'child_process';
+import * as net from 'net';
+import Agent from 'ssh-agent-js/client/index';
```

Comment:
this import looks wrong

Reply:
It does but as far as I understand npm it actually is right. 'Cause the package has two folders one for client and one for server. And the `main` field in `package.json` is `"main": "server/index.js"` so if you use:
```
import Agent from 'ssh-agent-js';
```
It tries to import the server instead of the client. Not sure whether there is another way to make it include the client instead.